Events/Ch24.event

```diff
@@ -0,0 +1,121 @@
+UNIT
+
+
+TurnBasedEvents:
+TurnEventPlayer(0x0,EventAfterExitingPrepScreen,1)
+END_MAIN
+
+CharacterBasedEvents:
+END_MAIN
+
+LocationBasedEvents:
+Armory(Ch24Armory,0,0)
+END_MAIN
+
+MiscBasedEvents:
+CauseGameOverIfLordDies
+AREA 0x0 EndingScene [0,0] [1,1]
+END_MAIN
+
+Dunno:
+//DO NOT TOUCH
+WORD $00
+
+Tutorial:
+//DO NOT TOUCH
+WORD $00
+
+TrapData:
+END_MAIN
+
+ALIGN 4
+
+BeginningScene:
+LOAD1 0x1 Ch24Enemies
+ENUN
+LOAD1 0x1 Ch24Enemies2
+ENUN
+LOAD2 0x0 Units
+ENUN
+ENDB
+
+EndingScene:
+MNCH 0x3e	//goes to where chapter 25 is in vanilla; let me know if we move ch25 somewhere else
+ENDA
+
+// Events
+EventAfterExitingPrepScreen:
+ENDA
+
+
+// Manual Movement
+
+// Scripted Fights
+
+// Units
+Ch24Enemies:
+UNIT CH24GenID Archer 0x00 Level(Ch24GenLvlT1, Enemy, 0) [8,15] 0x00 0x00 0x0 0x00 [SteelBow] GuardTileAI 
+UNIT CH24GenID Cavalier 0x00 Level(Ch24GenLvlT1, Enemy, 0) [12,11] 0x00 0x00 0x0 0x00 [SteelLance] ChaseOnceApproachedAI 
+UNIT CH24GenID Mercenary 0x00 Level(Ch24GenLvlT1, Enemy, 0) [11,16] 0x00 0x00 0x0 0x00 [SteelSword] AttackInRangeAI 
+UNIT CH24GenID Mercenary 0x00 Level(Ch24GenLvlT1, Enemy, 0) [9,17] 0x00 0x00 0x0 0x00 [SteelSword] AttackInRangeAI 
+UNIT CH24GenID Knight 0x00 Level(Ch24GenLvlT1, Enemy, 0) [10,9] 0x00 0x00 0x0 0x00 [SteelLance] AttackInRangeAI 
+UNIT CH24GenID Knight 0x00 Level(Ch24GenLvlT1, Enemy, 0) [4,9] 0x00 0x00 0x0 0x00 [SteelLance] AttackInRangeAI 
+UNIT CH24GenID Shaman 0x00 Level(Ch24GenLvlT1, Enemy, 0) [4,7] 0x00 0x00 0x0 0x00 [Flux] ChaseOnceApproachedAI 
+UNIT CH24GenID Shaman 0x00 Level(Ch24GenLvlT1, Enemy, 0) [12,14] 0x00 0x00 0x0 0x00 [Flux] NoAI 
+UNIT CH24GenID Shaman 0x00 Level(Ch24GenLvlT1, Enemy, 0) [12,19] 0x00 0x00 0x0 0x00 [Flux] NoAI 
+UNIT CH24GenID Shaman 0x00 Level(Ch24GenLvlT1, Enemy, 0) [7,9] 0x00 0x00 0x0 0x00 [Flux] NoAI 
+UNIT CH24GenID Shaman 0x00 Level(Ch24GenLvlT1, Enemy, 0) [14,17] 0x00 0x00 0x0 0x00 [Luna] NoAI 
+UNIT
+
+Ch24Enemies2:
+UNIT CH24GenID WyvernRider 0x00 Level(Ch24GenLvlT1, Enemy, 0) [11,0] 0x00 0x00 0x0 0x00 [SteelLance] NoAI 
+UNIT CH24GenID WyvernRider 0x00 Level(Ch24GenLvlT1, Enemy, 0) [12,0] 0x00 0x00 0x0 0x00 [SteelLance] NoAI 
+UNIT CH24GenID PegasusKnight 0x00 Level(Ch24GenLvlT1, Enemy, 0) [0,4] 0x00 0x00 0x0 0x00 [Javelin] AttackInRangeAI 
+UNIT CH24GenID PegasusKnight 0x00 Level(Ch24GenLvlT1, Enemy, 0) [1,4] 0x00 0x00 0x0 0x00 [Javelin] AttackInRangeAI 
+UNIT CH24GenID Druid 0x00 Level(Ch24GenLvlT2, Enemy, 0) [8,3] 0x00 0x00 0x0 0x00 [Flux, Eclipse] AttackInRangeAI 
+UNIT CH24GenID Bishop_F 0x00 Level(Ch24GenLvlT2, Enemy, 0) [12,6] 0x00 0x00 0x0 0x00 [Sleep, Shine, Physic] AttackInRangeAI 
+UNIT CH24GenID Bishop_F 0x00 Level(Ch24GenLvlT2, Enemy, 0) [1,9] 0x00 0x00 0x0 0x00 [Shine, Heal] NoAI 
+UNIT CH24GenID Sniper_F 0x00 Level(Ch24GenLvlT2, Enemy, 0) [8,10] 0x00 0x00 0x0 0x00 [Longbow] GuardTileAI 
+UNIT CH24GenID Knight_F 0x00 Level(Ch24GenLvlT2, Enemy, 0) [9,10] 0x00 0x00 0x0 0x00 [SilverLance] GuardTileAI 
+UNIT

```

Comment:
Is there no new unit yet?

Reply:
No, i haven't decided on the new unit yet.
If I can't decide I'll wait until a few more chapters are in and decide based on the available units.